ha-sip/src/mqtt.py

```diff
@@ -0,0 +1,51 @@
+# mqtt.py
+import paho.mqtt.client as mqtt
+from log import log
+from command_client import CommandClient
+
+class MQTTClient:
+    def __init__(self,
+                 broker_address: str,
+                 port: int,
+                 username: str,
+                 password: str,
+                 topic: str,
+                 command_handler: callable
```

Comment:
This should be ```command_handler: CommandHandler```.

If you have a circular import because of that, you can use
```
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from command_handler import CommandHandler
```
to solve that.